### ISSUE TITLE
Load pretrained model YOLO_small.ckpt  error

### ISSUE THREAD

**Issue by zihao-lu** [hizhangp/yolo_tensorflow#48]
When i restored the YOLO_small.ckpt，there was an error:         "Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?"

As we know, tensorflow saves model into four files  including checkpoint, index, data and meta, why this checkpoint only has one file, how can i use it? 
Thanks for anyone's help.

By the way, when i trained the model in Pascal VOC 2007  without using the pre-trained model, the loss couldn't decrease and changed between 8 and 12.

**Comment by hizhangp** [comment]
You should load the pretrained model in V1, please pay attention to the parameters in save(restorer) operation.

**Comment by zihao-lu** [comment]
thanks，i figured out this problem. The YOLO_small.ckpt was broken when I downloaded it. Finally I solved it..... Thanks a lot.

**Comment by zihao-lu** [comment]
I have another problem why the model is so hard to converge. Can you tell me some reasons? Thanks.
Have a good holiday.

**Comment by hizhangp** [comment]
Because it is a pretrained model, so it has already converged and cannot achieve a lower loss.

**Comment by zihao-lu** [comment]
hello, another problem, why you have  this layer 'net = slim.fully_connected(net, 512, scope='fc_33')'  
a  FC_layer with 512 output_size   it doesn't apear in the paper
